app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/model/Employee.kt

```diff
@@ -66,21 +65,17 @@ data class Employee(@field:Element(name = "geschlecht", required = false)
         val salutation = context.getString(resourceId)
         val salutationWithName = "$salutation $name $surname"
 
-        if (title.isEmpty()) {
+        if (title.isBlank()) {
             return salutationWithName
         } else {
             return "$salutationWithName, $title"
         }
     }
 
     override fun toString(): String {
-        var infoText = title
-
-        if (title != "") {
-            infoText = title + ' '
-        }
-
-        return infoText + name + ' ' + surname
+        return arrayOf(title, name, surname)
+                .filterNotNull()
```

Comment:
aren't you missing empty strings for titles then?

Reply:
Upon further inspection, I don’t believe that the string representation of `Employee` is used anywhere (we use `getNameWithTitle(Context)`). I’ve removed it now.